docs/modules/ROOT/pages/component-prometheus.adoc

```diff
@@ -0,0 +1,21 @@
+= Prometheus
+
+[abstract]
+The Couchbase Prometheus Server can be used to collect metrics from Couchbase Servers.
+
+== Overview
+
+https://prometheus.io/[Prometheus^] is a leading open-source monitoring solution. As a https://www.cncf.io/projects/prometheus/[graduated project^] of the Cloud Native Computing Foundation, it has become the industry standard for metrics collection and alert generation on cloud platforms.
```

Comment:
See the other PR but all sentences should be on their own single line. It is much easier to review then and has no impact on the output.